src/main/java/io/vertx/mqtt/impl/MqttClientImpl.java

```diff
@@ -1056,22 +1084,22 @@ private void handlePuback(int pubackMessageId) {
   }
 
   private void handlePubackTimeout(int packetId) {
-      ExpiringPacket expiredMessage;
-      synchronized (this) {
-        expiredMessage = qos1outbound.remove(packetId);
+    ExpiringPacket expiredMessage;
+    synchronized (this) {
+      expiredMessage = qos1outbound.remove(packetId);
 
-        if (expiredMessage == null) {
-          // the message has already been ACKed
-          log.debug("PUBLISH expiration timer fired but QoS 1 message has already been PUBACKed by server");
-          return;
-        }
-      }
-      countInflightQueue--;
-      Handler<Integer> handler = publishCompletionExpirationHandler();
-      if (handler != null) {
-        handler.handle(expiredMessage.packetId);
+      if (expiredMessage == null) {
+        // the message has already been ACKed
+        log.debug("PUBLISH expiration timer fired but QoS 1 message has already been PUBACKed by server");
+        return;
       }
     }
+    countInflightQueue--;
+    Handler<Integer> handler = publishCompletionExpirationHandler();
+    if (handler != null) {
+      handler.handle(expiredMessage.packetId);
+    }
+  }
```

Comment:
Are these changes related to this PR? It seems not, can you avoid them please?

Reply:
Its the indentation issue, corrected as per the standard.